beanstalk_worker/worker_start_build.py

```diff
@@ -194,16 +193,16 @@ def main():
                         is_build_running
 
                 if parent_build_running:
-                    logger.debug('Parents in build: %s, pushing job: %s back '
-                                 'to queue' % (parents_in_build, job_details))
+                    logger.info('Parents in build: %s, pushing job: %s back '
+                                'to queue' % (parents_in_build, job_details))
                     bs.use('master_tube')
                     bs.put(json.dumps(job_details))
                     bs.use('failed_build')
                 else:
-                    logger.debug(str(job_details))
+                    logger.info('Starting build for job: %s' % job_details)
                     result = start_build(job_details)
             else:
-                logger.debug("No job found to process looping again")
+                logger.info("No job found to process looping again")
```

Comment:
That is debatable, and I am not sure about it, TBH. We need to have a discussion at some point of time to decide what to log in which level. Here's an interesting read: http://stackoverflow.com/questions/7839565/logging-levels-logback-rule-of-thumb-to-assign-log-levels